stdlib.sh

```diff
@@ -814,7 +815,13 @@ layout_python() {
       return 1
     fi
 
-    VIRTUAL_ENV=$(direnv_layout_dir)/python-$python_version
+    if [[ -n $VIRTUAL_ENV ]]; then
```

Comment:
```suggestion
    if [[ -n "${VIRTUAL_ENV:-}" ]]; then
```
better handle `set -u`